.github/PULL_REQUEST_TEMPLATE/contributor_template.md

```diff
@@ -20,8 +20,6 @@
 
 <!-- Before requesting review, please ensure that your pull request fulfills the following requirements: -->
 
-- [ ] I have run the complete test suite to ensure all tests and linters pass.
-- [ ] I have thoroughly tested my code changes to ensure they work as expected, including adding additional tests for new functionality.
-- [ ] I have written clear and descriptive commit messages.
-- [ ] I have provided a detailed description of the changes in the pull request, including the problem it addresses, how it fixes the problem, and any relevant details about the implementation.
-- [ ] I have ensured that the code is well-documented and easy to understand.
+- [ ] I have run tests, verified functionality, and added tests for new features.
```

Comment:
```suggestion
- [ ] I have added tests for new features.
```

No need for them to run existing tests, CI will handle that... I frequently skip running tests locally myself!